app/src/main/kotlin/com/wire/android/ui/home/conversations/details/options/GroupConversationOptions.kt

```diff
@@ -11,32 +13,179 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
 import com.wire.android.R
 import com.wire.android.model.Clickable
+import com.wire.android.ui.common.WireDialog
+import com.wire.android.ui.common.WireDialogButtonProperties
+import com.wire.android.ui.common.WireDialogButtonType
+import com.wire.android.ui.home.conversations.details.GroupConversationDetailsViewModel
+import com.wire.android.ui.home.conversationslist.common.FolderHeader
+import com.wire.android.ui.theme.WireDimensions
 import com.wire.android.ui.theme.wireColorScheme
 
+@Composable
+fun GroupConversationOptionsScreen(
```

Comment:
I don't know if it impacts anyhow but shouldn't the class name be the same as the file name? Or it only impacts the full classes and not in compose?

Reply:
i did update the functions names to match the file